docs/src/main/tut/datatypes/eithert.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: docs
+title:  "EitherT"
+section: "data"
+source: "core/src/main/scala/cats/data/EitherT.scala"
+scaladoc: "#cats.data.EitherT"
+---
+# EitherT
+
+`Either` can be used for error handling in most situations. However, when
+`Either` is placed into effectful types such as `Option` or`Future`, a large
+amount of boilerplate is required to handle errors. For example, consider the
+following program:
+
+```tut:book
+import scala.util.Try
+import cats.implicits._
+
+def parseDouble(s: String): Either[String, Double] =
+  Try(s.toDouble).map(Right(_)).getOrElse(Left(s"$s is not a number"))
```

Comment:
Super nitpicky and unreasonable comment: `s` could still technically be a number that can't be represented as a `Double`. It might be best to leave it simple and just say "`$s is not a valid double`.